src/screens/SettingsScreen.tsx

```diff
@@ -46,6 +47,12 @@ export const SettingsScreen = () => {
             action={() => navigate('Settings/Wearables')}
           />
           <Divider />
+          {__DEV__ && process.env.NODE_ENV !== 'test' && (
+            <>
+              <ToggleEnvironmentMenuItem />
+              <Divider />
+            </>
+          )}
```

Comment:
Whether instead of this or in addition to it, might consider putting this on the LoginScreen so devs can switch envs without having to login.

Reply:
It is a good idea but I'm going to kick that down the road for now since I already built the component for the settings screen.